src/filters/dasher.c

```diff
@@ -2865,6 +2866,26 @@ static void dasher_add_descriptors(GF_List **p_dst_list, const GF_PropertyValue
 	}
 }
 
+static void dasher_add_inband_event(GF_DashStream *ds)
+{
+	GF_MPD_Inband_Event *nielsen_event;
+	GF_MPD_Inband_Event *custom_event;
+	if(ds->stream_type == GF_STREAM_AUDIO) {
+		GF_SAFEALLOC(custom_event, GF_MPD_Inband_Event);
+		custom_event->scheme_id_uri = gf_strdup("https://aomedia.org/emsg/ID3");
```

Comment:
Is there an issue tracking the generalization of this flag?

The `scheme_id_uri` and `value` could be extracted from the `GF_ID3_TAG` properties attached to the audio packets.

Reply:
That's a good idea. Do you feel like doing it before merging or should we open a separate issue?

Reply:
Better to open a separate issue. We can try to solve it at a later time. @sebastianBD95 @gomezpirry